CONTRIBUTING.md

```diff
@@ -0,0 +1,43 @@
+# Contributing
+
+Welcome! Thanks for your interest in contributing to QUnit. This document is
+intended to help make your experience smooth and enjoyable, so if you feel
+anything should be added or clarified definitely let us know.
```

Comment:
Nitpick: With this sentence structure, I would love to see a comment after "added or clarified". That said, the sentence also feels a bit long and discount- perhaps the clause starting with "if you feel" could be its own sentence.